CreateTablesScript.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE IF NOT EXISTS `mydb`.`USERS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `email` VARCHAR(45) NOT NULL,
+  `username` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(45) NOT NULL,
+  `gender` INT NOT NULL,
+  `birthdate` DATE NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
+  UNIQUE INDEX `password_UNIQUE` (`password` ASC) VISIBLE,
+  UNIQUE INDEX `login_UNIQUE` (`username` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`PARAMETERS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`PARAMETERS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `user_id` INT NOT NULL,
+  `weight` FLOAT NOT NULL,
+  `height` FLOAT NOT NULL,
+  `physical_acitvity` INT NOT NULL,
+  `target_weight` FLOAT NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
+  UNIQUE INDEX `user_id_UNIQUE` (`user_id` ASC) VISIBLE,
+  CONSTRAINT `user_id_fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `mydb`.`USERS` (`id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`INGREDIENTS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`INGREDIENTS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `calories` INT NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`DISHES`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`DISHES` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NOT NULL,
+  `description` VARCHAR(500) NOT NULL,
+  `category` INT NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`PRIORITY`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`PRIORITY` (
+  `user_id` INT NOT NULL,
+  `dish_id` INT NOT NULL,
+  `priority` INT NOT NULL DEFAULT 10,
+  PRIMARY KEY (`user_id`, `dish_id`),
+  INDEX `dish_id_idx` (`dish_id` ASC) VISIBLE,
+  CONSTRAINT `user_id_fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `mydb`.`USERS` (`id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `dish_id_fk`
+    FOREIGN KEY (`dish_id`)
+    REFERENCES `mydb`.`DISHES` (`id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`PRODUCTS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`PRODUCTS` (
+  `dish_id` INT NOT NULL,
+  `ingredient_id` INT NULL,
+  `weight` VARCHAR(45) NOT NULL,
```

Comment:
Let's make it float. It's easier to calculate - no need to parse strings and cast to numbers.

Reply:
Random mistake:)